service/controller/key/key.go

```diff
@@ -49,6 +49,8 @@ const (
 
 	AnnotationEtcdDomain        = "giantswarm.io/etcd-domain"
 	AnnotationPrometheusCluster = "giantswarm.io/prometheus-cluster"
+	AnnotationNodePoolMinSize   = "cluster.k8s.io/cluster-api-autoscaler-node-group-min-size"
```

Comment:
We can do it after merging this PR if you prefer. But it'd be cool to have these annotations in [the apiextensions `pkg` package](https://github.com/giantswarm/apiextensions/tree/master/pkg/annotation) like we do with others.

Reply:
Yeah I agree but we are still on a very old version of apiextensions and I don't know if that is going to change soon.

Reply:
Why don't we get the label directly from the CAPI package, as we do with the cluster ID label?

https://github.com/giantswarm/azure-operator/blob/416215e88c0054035ee621bf5235e1650580276a/service/controller/resource/clusterownerreference/resource_test.go#L44

Reply:
Yeah that's what Jose asked for.
Requires us to bump apiextensions to 2.2.0.

Reply:
In my example we import the label from `sigs.k8s.io/cluster-api/api/v1alpha3`. Is updating `apiextensions` required for using that?

Reply:
I didn't find that constant in the CAPI repo. Did you find it?

Reply:
Ah my bad, I just assumed they were there. I now looked around couldn't find them, either 😞

Reply:
this is done